aptos-move/framework/aptos-framework/sources/aptos_governance.move

```diff
@@ -53,10 +54,18 @@ module aptos_framework::aptos_governance {
     const EMETADATA_HASH_TOO_LONG: u64 = 10;
     /// Account is not authorized to call this function.
     const EUNAUTHORIZED: u64 = 11;
+    /// The stake pool is using voting power more than it has.
+    const EVOTING_POWER_OVERFLOW: u64 = 12;
+    /// Partial voting feature hasn't been properly initialized.
+    const EPARTIAL_VOTING_NOT_INITIALIZED: u64 = 13;
+    /// The proposal in the argument is not a partial voting proposal.
+    const ENOT_PARTIAL_VOTING_PROPOSAL: u64 = 14;
 
     /// This matches the same enum const in voting. We have to duplicate it as Move doesn't have support for enums yet.
     const PROPOSAL_STATE_SUCCEEDED: u64 = 1;
 
+    const MAX_U64: u64 = 18446744073709551615;
```

Comment:
I wonder if we can just move this into the math_u64 module and add a max_value function or something. @gerben-stavenga what do you think?

Reply:
Sounds good. Since this refactor could affect many places, I will do it in a later PR. Created an issue: https://github.com/aptos-labs/aptos-core/issues/7894